repo/raja-perf/application.py

```diff
@@ -10,18 +10,68 @@
 
 class RajaPerf(ExecutableApplication):
     """RAJA Performance suite"""
+
     name = "raja-perf"
 
-    tags = ['asc','single-node','sub-node','structured-grid',
-            'atomics','simd','vectorization','register-pressure',
-            'high-memory-bandwidth','regular-memory-access',
-            'mpi','network-point-to-point','network-latency-bound',
-            'c++','raja','cuda','hip','openmp','sycl']
+    tags = [
+        "asc",
+        "single-node",
+        "sub-node",
+        "structured-grid",
+        "atomics",
+        "simd",
+        "vectorization",
+        "register-pressure",
+        "high-memory-bandwidth",
+        "regular-memory-access",
+        "mpi",
+        "network-point-to-point",
+        "network-latency-bound",
+        "c++",
+        "raja",
+        "cuda",
+        "hip",
+        "openmp",
+        "sycl",
+    ]
+
+    executable(
+        "run",
+        template=["{execute}" + " --size {size}" + " --repfact {repfact}"],
+        use_mpi=True
+    )
+
+    # executable(
```

Comment:
Should the commented out code be removed?

Reply:
It has been removed.